src/main/java/mate/jdbc/service/ManufacturerService.java

```diff
@@ -0,0 +1,19 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Manufacturer;
+
+@Service
+public interface ManufacturerService {
+
```

Comment:
remove the first empty line in all classes/interfaces/methods where it is present
```suggestion
```